kbase-extension/static/kbase/css/kbaseNarrative.css

```diff
@@ -17,35 +17,177 @@
     margin: 2em 4.5em;
 }
 
-#kb-data-staging-table_wrapper{
-    margin-top: 15px;
+
```

Comment:
thus begins where i reworked a lot of the css class names and structure for the staging data table